inc/modules/export/epub/class-epub201.php

```diff
@@ -2034,12 +2050,22 @@ protected function fetchAndSaveUniqueImage( $url, $fullpath ) {
 		}
 
 		if ( $this->compressImages ) {
-			$format = explode( '.', $filename );
-			$format = strtolower( end( $format ) ); // Extension
-			try {
-				\Pressbooks\Image\resize_down( $format, $tmp_file );
-			} catch ( \Exception $e ) {
-				return '';
+			/**
```

Comment:
Can you move this filter into the contsructor? Set `$this->compressImages = false` next to this code:

https://github.com/pressbooks/pressbooks/blob/dev/inc/modules/export/epub/class-epub201.php#L403

So that it will be easier to spot what's going on if ever there's an issue.

Reply:
Hmmm, I see your point, but then it isn't the same filter anymore. How I implemented the filter, you can turn of compression on a per image base and that is needed here, as I don't want to turn of compression for all the images.